api-docs.html

```
@@ -188,7 +188,7 @@ <h2 id="quickstart">Quick Start — First API Call in 60 Seconds</h2>
     headers: { 'X-API-Key': API_KEY }
   });
   const data = await resp.json();
-  data.forEach(adv => console.log(`[${adv.severity}] ${adv.title} — CVSS:${adv.cvss_score}`));
+  data.items.forEach(adv => console.log(`[${adv.severity}] ${adv.title} — CVSS:${adv.cvss_score}`));
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)